hack/install-go.sh

```diff
@@ -4,8 +4,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-wget https://go.dev/dl/go1.21.4.linux-amd64.tar.gz
+wget -q https://go.dev/dl/go1.21.4.linux-amd64.tar.gz
 rm -rf /usr/local/go && tar -C /usr/local -xzf go1.21.4.linux-amd64.tar.gz
```

Comment:
In future, can we not update go for the developer user machine like this? Many users have set up go for various projects and an individual project should not potentially globally change the installed go version, which could affect other projects builds. I think this script may only be applicable in CI so maybe this is OK, but generally I prefer tooling to be project specific, I've hit too many cross-project tooling issues to like projects managing global tooling like this.

Reply:
Sure, in future, we can consider using devbox and the like to have a virtual environment that's project specific. Although, that'd be out of scope for this PR.

Reply:
Let's make it in separate PR.

Reply:
👍

Reply:
👍